extensions/amp-brid-player/validator-amp-brid-player.protoascii

```diff
@@ -26,12 +26,8 @@ tags: {  # amp-brid-player
   }
   attr_lists: "common-extension-attrs"
 }
-tags: {  # <amp-brid-player>
-  html_format: AMP
-  tag_name: "AMP-BRID-PLAYER"
-  disallowed_ancestor: "AMP-SIDEBAR"
-  requires_extension: "amp-brid-player"
-  attrs: { name: "autoplay" }
+attr_lists: { # <amp-brid-player> attributes that are common to both AMP & A4A.
```

Comment:
It looks like the only reason to split all of these between AMP docs and AMP Ads documents is because of `fullscreen-on-landscape`.

Is there no way to allow `fullscreen-on-landscape` for AMP Ads?

Further, if `autoplay` is now available to all of these, perhaps that should be moved into the `common` for each `tags` and then it'd only be necessary to add `fullscreen-on-landscape` as an attribute for `html_format: AMP`.

Nit (but more than a nit for consistency): two spaces between `{` and `#`.